specification/eventgrid/Azure.Messaging.EventGrid.SystemEvents/Microsoft.Communication/AzureCommunicationServices.tsp

```diff
@@ -1227,13 +1227,13 @@ model AcsMessageMediaContent {
   /** Optional. The caption for the media object, if supported and provided */
   caption?: string;
 
-  /** Optional. Set to  true  if the sticker is animated; false otherwise. */
+  /** Optional. Set to  true  if the sticker is animated;  false  otherwise. */
```

Comment:
There is an "NBSP" space around "true" and "false". It leads to the code generated like this. Please just delete the spaces and add them back manually in your text editor.

![image](https://github.com/user-attachments/assets/2c9d9b01-adc4-40ef-b457-7e244e87a65f)